src/avram/tasks/db/schema_restore.cr

```diff
@@ -0,0 +1,13 @@
+class Db::Schema::Restore < LuckyCli::Task
+  summary "Restore database from a sql dump file"
+
+  def initialize(@import_file : String? = nil, @quiet : Bool = false)
+  end
+
+  def call
+    import_file = @import_file || ARGV.first? || raise "An import file must be provided"
```

Comment:
Maybe "An import  file *path*" so it is clear it is the path and not file contents from STDIN or something